tests/components/autocomplete_dropdown.test.ts

```diff
@@ -343,6 +368,14 @@ describe("Autocomplete parenthesis", () => {
 });
 
 describe("composer Assistant", () => {
+  beforeEach(async () => {
+    genericBeforeEach();
+  });
+
+  afterAll(() => {
+    restoreDefaultFunctions();
+  });
```

Comment:
the only thing that counts is that the registry is back to normal when we get out of this test file. And the issue is that some tests require a different registry than  the one defined in the genericBeforeEach, like in `autocomplete boolean functions`  but I guess we can just add them to the list of existing functions (since we actually already added them in the real functionRegistry >< )